### ISSUE TITLE
Bug (0.9.25) PathItem.extend

### ISSUE THREAD

**Issue by poofik** [paperjs/paper.js#837]
line 7315
```javascript
from = index + amount - 1 === total ? index - 1 : index,
```
causes -1 and _path = undefined in _adjustCurves, can we change it back to 
```javascript
from = index > 0 ? index - 1 : index,
```

**Comment by sapics** [comment]
I guess that it would be fixed in #816.
If you need fixed one, you can download it from http://paperjs.org/download/paperjs-nightly.zip

**Comment by lehni** [comment]
Yes this should indeed be addressed. New release coming up very soon!
